api/src/pcapi/core/educational/adage_backends/adage.py

```diff
@@ -25,6 +25,9 @@ def notify_prebooking(self, data: prebooking.EducationalBookingResponse) -> mode
             data=data.json(),
         )
 
+        if api_response.status_code == 404:
+            return models.AdageApiResult(sent_data=data, response=dict(api_response.json()), success=True)  # type: ignore [arg-type]
+
         if api_response.status_code != 201:
```

Comment:
(détail / style / question de goût) On pourrait écrire : 

```suggestion
        if api_response.status_code not in (201, 404):
```

Reply:
Ah, et au passage j'ajouterais bien un commentaire, parce que considérer une 404 comme acceptable, ça ne nous paraîtra peut-être pas évident dans 6 mois. :) Suggestion : 

> Adage returns a 404 if the institution does not have any email.

Reply:
Hmm, au fait, est-ce que Adage renvoie aussi une 404 dans les mêmes conditions pour les autres fonctions de notification plus bas ?

Reply:
Justement je viens de mettre à jour avec un autre cas, j'ai vérifié avec Mickael :)